### PR TITLE
NIAD-1388: Object storage documentation

### PR DESCRIPTION
**Pull request by ElliotB-96**

### CODE REVIEW

File: OPERATING.md [hunk 1]
```diff
@@ -28,7 +28,16 @@ yyyy-MM-dd HH:mm:ss.SSS Level=INFO Logger=u.n.a.g.e.m.AgentDirectoryExtractor Co
 
 ## Object Storage (AWS S3 / Azure Blob)
 
-TODO: Document storage scheme
+- Data stored:
+  - The patients transformed structured record once it has been transformed into an ehrExtract that is ready to send to MHS Outbound
+  - Attachments associated with the patients structured record once they have been transformed into an outbound message for sending to MHS Outbound
+- Filename convention:
+  - EhrExtracts files are named as a concatonation of {conversationId}/{conversationId}.json
```
Comment (michael-kainos) [inline OPERATING.md:35]: change concatonation to concatenation

Comment (michael-kainos) [inline OPERATING.md:35]: is it correct that we use the same conversationId twice as the file names?

**Review by michael-kainos — COMMENTED** [review]
There is not documentation about the configuration of an object store, should we add this?

File: OPERATING.md (hunk 1 above)
Comment (ElliotB-96) [reply OPERATING.md:35]: Yes, for the transformed structured record

**Comment by ElliotB-96** [comment]
> There is not documentation about the configuration of an object store, should we add this?

There is configuration documentation for the storage in the readme.md

**Review by michael-kainos — APPROVED** [review]

**Review by bbell-kainos — APPROVED** [review]

**Review by rorysproule1 — APPROVED** [review]

**Pull request merged**